core/runtime/execute_engine.cpp

```diff
@@ -163,26 +157,27 @@ std::vector<at::Tensor> execute_engine(std::vector<at::Tensor> inputs, c10::intr
 
     for (size_t o = inputs.size(); o < (compiled_engine->num_io.first + compiled_engine->num_io.second); o++) {
```

Comment:
Same thing here, use output_binding_names to pull out the TensorRT binding index / address